templates/slim/helpers.rb

```diff
@@ -31,6 +31,38 @@ def to_boolean val
     val && val != 'false' && val.to_s != '0' || false
   end
 
+  ##
+  # This function is from the asciidictor-html5s project
+  # Copyright 2014-2017 Jakub Jirutka <jakub@jirutka.cz> and the Asciidoctor Project.
+  # Licensed under the MIT License.
```

Comment:
It’s not needed to copy copyright here; both projects are MIT licensed and contain Asciidoctor project as one of the copyright holders.

Reply:
Fixed